python/versionChecks.py

```diff
@@ -0,0 +1,92 @@
+import functools
+
+from vector.enums import ENVIRONMENT_VERSION_TYPE_T
```

Comment:
Does this exist in older version of dataAPI, I thought Richard just added this.

Reply:
Richard added `REVISION_2024_CODED_MOCK_DATA` to `ENVIRONMENT_VERSION_TYPE_T`, but `ENVIRONMENT_VERSION_TYPE_T` has existed for a while.